app/Filament/Clusters/Settings/Resources/SettingResource/Pages/ManageSettings.php

```diff
@@ -13,7 +16,26 @@ class ManageSettings extends ManageRecords
     protected function getHeaderActions(): array
     {
         return [
-            Actions\CreateAction::make(),
+            Action::make('apply')
+                ->label('Apply Settings')
+                ->requiresConfirmation()
+                ->action($this->setConfigFromDatabase()),
         ];
     }
+
+    protected function setConfigFromDatabase(): void
```

Comment:
This function should work and update the config from the Settings page only it does not

Reply:
Remove the function, then [adjust/ restore the static methods in the Settings Model](https://github.com/pelican-dev/panel/blob/main/app/Models/Setting.php#L32-L84).
The [SettingsServiceProvider](https://github.com/pelican-dev/panel/blob/main/app/Providers/SettingsServiceProvider.php#L55-L106) takes care about applying the settings. You will probably need to update it as well.